crates/rspack_core/src/compiler/module_executor/execute.rs

```diff
@@ -72,6 +73,50 @@ impl Task<MakeTaskContext> for ExecuteTask {
     compilation.plugin_driver = compilation.buildtime_plugin_driver.clone();
 
     let id = EXECUTE_MODULE_ID.fetch_add(1, std::sync::atomic::Ordering::Relaxed);
+    let errors = compilation
+      .make_artifact
+      .diagnostics()
+      .into_iter()
+      .filter(|d| matches!(d.severity(), RspackSeverity::Error))
+      .collect_vec();
```

Comment:
There doesn't seem to be any consumption of `errors` vec, maybe we can use `.any()` instead of `.filter()` + `.collect_vec()`